src/lib/serviceRoutinesV2/postEntity.cpp

```diff
@@ -100,14 +100,14 @@ std::string postEntity
     }
   }
 
-  // Default value for status code: SccCreated
+  // Default value for status code: SccNoContent
   if ((ciP->httpStatusCode == SccOk) || (ciP->httpStatusCode == SccNone) || (ciP->httpStatusCode == SccCreated))
```

Comment:
Not sure of all the || clauses in this if... Is the last one (for SccCreated) really needed?